coverage-report/pom.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-  ~ Copyright (c) 2010-2022. Axon Framework
+  ~ Copyright (c) 2010-2023. Axon Framework
```

Comment:
That would be great, but indeed better done in a separate pr. There might be some additional configuration needed for proper reports since the `spring-boot-3-integrationtests` on purpose doesn't have the axon parent module as parent.

Reply:
Great idea. I've described our intent to adjust Sonar to point to JDK17, and to include the Spring Boot 3 tests, in issue #2561.